benchmarks/backend_request_func.py

```diff
@@ -27,8 +27,8 @@ class RequestFuncInput:
 class RequestFuncOutput:
     generated_text: str = ""
     success: bool = False
-    latency: float = 0
-    ttft: float = 0  # Time to first token
+    latency: float = 0.0
+    ttft: float = 0.0  # Time to first token
```

Comment:
so unfortunate mypy cannot handle this :(

Reply:
On the other hand mypy hints a potential source for an actual bug - I think in this context any plausible use of that int should however also be compatible with int or float.
E.g. `int(str(0))` works but `int(str(0.0))` breaks.